### PR TITLE
Created mobile setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Theoneste1**
Created mobile setup and mobile version skeleton
Used a Figma design to create a UI.
Used the Flexbox to place elements on the page.
Build a personal portfolio site as it has been designed.
Used the images and backgrounds to enhance the look of the website.
set up a new repository and prepare it for development using best practices of linters and everything is working fine.
built the first 2 sections of the mobile website using the template  I chose  from the given different examples

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <h1>I'm Theoneste Glad to see you!</h1>
+<p class="prg">
+    I’m a software developer! I can help you build a product ,
+     feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded, don’t 
+      hestiate to contact me.
+</p>
+
+<h4 class="connect">LET'S CONNECT</h4>
+<ul class="ul-list-icons">
```
Comment (danielufeli) [inline index.html:25]: - [ ] Nice working on the social icons, kindly add (`a` tag) links to the icons
- [ ] Kindly re-size and align the social icons so they have the same height and width.
- [ ] Kindly use the same `font` provided in the Figma Template, so it aligns with what we have in the Figma Template

### Figma Template
fig1

### Your Design
my_1

**Review by danielufeli — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <li><img src="./photos/icons/twitter-icon.png" alt=""></li>
```
Comment (danielufeli) [inline index.html]: - [x] Nice working on the social icons, kindly add (`a` tag) links to the icons
- [x] Kindly re-size and align the social icons so they have the same height and width.
- [x] Kindly use the same `font` provided in the Figma Template, so it aligns with what we have in the Figma Template

### Figma Template
fig1

### Your Design
my_1

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Theoneste1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Clean Design ✔️
Passing linters ✔️
Descriptive Readme ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Theoneste1) [reply index.html]: @danielufeli , Hi, I would like to let you know that I already implemented the your feedbacks

1. Worked on the social icons, added (a tag) links to the icons
2.  3. re-sized and aligned the social icons so they have the same height and width.
4. Used the same font provided in the Figma Template, so it aligns with what we have in the Figma Template.
pic

File: style.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+}
```
Comment (mukhammadrizooff) [inline style.css:7]: - [ ] [OPTIONAL] Good job so far! Your project opens only the same size in the mobile version, resulting in a white field on the right when expanded from 300 px to 800 px. My advice to you would be to ask you to fix this problem. Let's learn [additional information](https://youtu.be/yU7jJ3NbPdA).

image

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Theoneste1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- All linters passed 💯 
- Correct use of Gitflow 💯 
- Descriptive messages 💯
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**